Added f to format string

uwagura opened the pull request:
<!--

Thanks for your contribution! Please:
* List any related issues with a "closes" or "addresses" tag.
* Add a helpful title & description.
* Complete the checklist.

-->
Closes #6331 . Test not needed because I literally just added a character
**Check List**

- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Applied any dependency changes to both `setup.cfg` (and `conda-environment.yml` if present).
- [x] Tests are included (or explain why tests are not needed).
- [x] Changelog entry included if this is a change that can affect users
- [x] [Cylc-Doc](https://github.com/cylc/cylc-doc) pull request opened if required at cylc/cylc-doc/pull/XXXX.
- [x] If this is a bug fix, PR should be raised against the relevant `?.?.x` branch.

oliver-sanders commented:
Hi @uwagura and welcome to the Cylc community :tada:

Thanks for your bug report and many thanks for taking the time to fix the issue.

For legal reasons, we need all commit authors to enter their names into the [CONTRIBURINTING.md](https://github.com/cylc/cylc-flow/blob/master/CONTRIBUTING.md) file (no matter how small the fix) in order to agree to the [CLA](https://github.com/cylc/cylc-flow/blob/master/CONTRIBUTING.md#contributor-licence-agreement-and-certificate-of-origin).

Make sure you write your name in the same format it appears in your commit (run `git shortlog -se` and copy your name as it appears there, or add an entry into the `.mailmap` file if you want it to appear differently).

uwagura commented:
@oliver-sanders , feels a bit strange to call myself a contributor for such a small change, but I've gone ahead and added myself to the contributors list and mailmap. I also added an entry to the changelog using towncrier, though I'm not entirely sure if I did it correctly. Let me know how how everything looks

oliver-sanders commented:
It's a legal thing we have to do even with the smallest commits, we only need to do it once.

We greatly appreciate bugfixes, thanks for your contribution.
